### DISCUSSION TITLE
Filter specific subdirectories

### DISCUSSION THREAD

**Discussion by ricdeters** [WinMerge/winmerge#1146] [General]
Is it possible to filter for specific subdirectories?

d: \\myfolder1\myfolder2\*$

I have tried, with no success, to limit which subdirectories are compared, but all subdirectories seems to be the only way to do it.

**Comment by sdottaka** [comment]
The file filter pattern must be a regular expression. If you change the pattern as shown below, you will get the expected result.
~~~
d: \\myfolder1\\myfolder2.*$
~~~

![image](https://user-images.githubusercontent.com/98126/150609247-fb7e4310-3d84-4caa-8d3d-0c1b5a8bd35e.png)
